File: 100-Days-Series/Day 97/PalindromeArray.js

// Function to check if an array is a palindrome.
const getCheckPalindrome=(arr)=>{
    let start=0;
    let end=arr.length-1;
    while(start<end){
        if(arr[start]!=arr[end]){
            return false;
        }
        start++;
        end--;
    }
    return true;
}
// Let's Start Here
const arr=[1,2,3,2,1];
console.log("Array : "+arr);
result=getCheckPalindrome(arr);
if(result){
    console.log("The Array is a Palindrome.");
}else{
    console.log("The Array is not a Palindrome.");
}

File: 100-Days-Series/Day 97/PalindromeArray.py

# Function to check if a array is a palindrome.
def get_check_palindrome(arr):
    start=0
    end=len(arr)-1
    while start<end:
        if arr[start]!=arr[end]:
            return False
        start+=1
        end-=1
    return True

# Let's Start Here
arr=[1,2,3,2,1]
print(f'Array : {arr}')
result=get_check_palindrome(arr)
if result:
    print('The Array is a palindrome.')
else:
    print('The Array is not a palindrome.')
    
